internal/events/eventer.go

```diff
@@ -280,6 +302,12 @@ func (e *Eventer) Register(
 		topic,
 		e.webhookSubscriber,
 		func(msg *message.Message) error {
+			if builtAt := msg.Metadata.Get(BuiltKey); builtAt != "" {
```

Comment:
I think this is okay, but we could also add it at line 141 in the handler chain for the router.  (I think I have a slight preference for that -- it will tell us when we're taking the message off the queue, before we dead-letter, retry, etc.) On the other hand, you'll have to worry about conflicting with @teodor-yanev 's #1994 , so I can see a tactical argument for doing it here.

Reply:
I moved it to where you suggested. If there are merge conflicts they should be easy enough to fix.